pkg/service/v0/errorcode/errorcode.go

```diff
@@ -0,0 +1,75 @@
+package errorcode
```

Comment:
Is this package really specific to this service package? Otherwise move it to pkg.

Reply:
It is specific to MS Graph